korman/exporter/convert.py

```diff
@@ -522,6 +559,11 @@ def _(temporary, parent):
             return temporary
 
         def do_pre_export(bo):
+            if bo.type == "ARMATURE":
+                so = self.mgr.find_create_object(plSceneObject, bl=bo)
```

Comment:
I realize we can't do everything in one go, but I am thinking toward the future where we can (hopefully) export avatar animations from Korman, and IIRC avatar animation PRPs don't have a scene object in them at all.